pylint/checkers/exceptions.py

```diff
@@ -112,13 +112,18 @@ def _is_raising(body: list[nodes.NodeNG]) -> bool:
     "W0702": (
         "No exception type(s) specified",
         "bare-except",
-        "Used when an except clause doesn't specify exceptions type to catch.",
+        "A bare ``except:`` clause will catch ``SystemExit`` and ``KeyboardInterrupt``"
+        " exceptions, making it harder to interrupt a program with ``Control-C``, and "
```

Comment:
I don't really like the mixing of spaces at the end of the string and at the beginning of the string. Could you make that consistent?